docs/claims.md

```diff
@@ -0,0 +1,194 @@
+---
+id: claims
+title: Polkadot Claims
+sidebar_label: Claims
+---
+
+If you made a DOT purchase in one of the pre-launch sales, then you will need to claim your DOT
+tokens. For those who made a pre-genesis claim by sending a claim transaction to the Claims Contract
+on Ethereum, you will still need to send a free _attest_ transaction that will agree to the terms
+and conditions of your allocation. If you did not make a pre-genesis claim, then you will claim and
+attest in a single transaction.
+
+This guide will walk you through the steps for either making a claim or attesting to the statement.
+
+If you are making a claim on Polkadot for the first time please read on below in the
+[making a claim](#making-a-claim) section. If you've already claimed during the pre-genesis claims
+please proceed to [attesting to a statement](#attesting-to-a-statement) section instead.
+
+## Making a Claim
+
+If you did not make a claim in the pre-genesis claims period, then you are able to claim your DOT
+tokens after genesis. There is no time limit for making your claim so feel free to do this whenever
+you are most comfortable.
+
+> Note: When you make a claim, you will also attest to the agreement that corresponds to your DOT
+> allocation. The two actions "claim" and "attest" are done in a single transaction, but for the
+> most part this is simplified from the perspective of the UI.
+
+### What You Will Need
+
+- The Ethereum account that holds the DOT indicator tokens
+- The MyCrypto wallet
+- A Polkadot account
+
+You should already have your Ethereum account that holds the DOT indicator tokens from Polkadot's
+prior sales. You will need to have access to this account in order to make a signature.
+
+[MyCrypto][] is a versatile wallet that supports a variety of storage methods for your Ethereum
+account. Go to their downloads page and ensure that you download the latest version for your
+operating system. This is important because the latest version will always have the latest security
+patches.
+
+> **NOTICE**: It is much more secure to download and use the MyCrypto app locally. You can always
+> find the most up-to-date releases of the desktop app on their [releases page][mycrypto].
+
+You will need a Polkadot account to which to fulfill your DOTs claim. Please follow the instructions
+on the [account generation][] page for generating a new Polkadot account.
+
+#### Claiming your DOT with MyCrypto
+
+The Polkadot JS [Claims app][] helps you sign a message from MyCrypto. MyCrypto is a good choice in
+case you have stored the key to the Ethereum account holding your DOT indicator tokens on a hardware
+device like a Ledger Nano S or a Trezor. It also supports raw private keys, mnemonics, and Parity
```

Comment:
MyCrypto with Trezor can't sign, apparently.  Not sure why.
<img width="248" alt="Screen Shot 2020-05-26 at 4 38 57 PM" src="https://user-images.githubusercontent.com/1417717/82914182-7917a380-9f6f-11ea-977b-0ed1ce4d2663.png">

Reply:
I wonder if this is a new change?